src/Assets/LightSkyDefense/Scripts/Enemy.cs

```diff
@@ -27,6 +30,7 @@ void Update()
 
         if (health<=0)
```

Comment:
waarom word dit elke frame gecheked in plaats van alleen wanneer de health word geupdate

Reply:
Fixed in latest commit